scripts/generate-test-services.ts

```diff
@@ -109,3 +109,10 @@ generateTestServicesPackage(packageOutputDir, 'v2');
 generateTestServicesPackage(packageOutputDir, 'v4');
 generateTestServicesWithLocalCoreModules(coreUnitTestOutputDir, 'v2');
 generateTestServicesWithLocalCoreModules(coreUnitTestOutputDir, 'v4');
+
+generate({
```

Comment:
Cloud you perhaps wrap this generate also in a function with a nice name like `generateTestServiesE2Etests` or something like this.

Reply:
Or perhaps it makes sense to split the generation in multiple tasks see comment above.

Reply:
I'm a bit afraid of adding more scripts to the main `package.json`. But I see the use case, since currently, we are touching the v4 and e2e parts. Let's add 3 additional scripts then. I hope we can remove them at the end of the quarter.